source_files/edge/rad_defs.h

```diff
@@ -30,6 +30,13 @@ struct rts_state_s;
 struct rad_script_s;
 struct rad_trigger_s;
 
+typedef enum
+{
+	RTS_TAG_NUMBER = 0,
+	RTS_TAG_HASH
+}
+s_tagtype_e;
```

Comment:
I am not sure what the s_ prefix means here?  Maybe `rad_tagtype_e`?

It is also a bit of a different style than other enums seeing, EDGE could use a style guide, it is pretty consistent in places, and inconsistent in others.  
```typedef enum
{
	ga_nothing = 0,
	ga_newgame,
	ga_loadlevel,
	ga_loadgame,
	ga_savegame,
	ga_intermission,
	ga_finale,
	ga_endgame
}
gameaction_e;
```

Reply:
I agree about the style stuff; I had noticed that a lot of RTS parsing structures had an s_ prefix (s_enabler_t, etc), so I threw that in there for now. I think it should suffice for this PR but there really